### PR TITLE
[Attestation] Enable dcap logging callback in oeutil for windows

### PR DESCRIPTION
**Pull request by RRathna**
This brings parity in oeutil between linux and windows.

### CODE REVIEW

**Comment by RRathna** [comment]
bors try

File: tools/oeutil/host/generate_evidence.cpp [hunk 1]
```diff
@@ -221,11 +221,9 @@ static void oeutil_quote_provider_log(
 // Set DCAP client (libdcap_quoteprov) log callback
 static void set_log_callback()
 {
-#if defined(__linux__)
     // Initialize quote provider and set log callback
     oe_initialize_quote_provider();
     oe_sgx_set_quote_provider_logger(oeutil_quote_provider_log);
```
Comment (mingweishih) [inline tools/oeutil/host/generate_evidence.cpp:226]: Should the oeutil_quote_provider_log be updated as well?

Comment (RRathna) [reply tools/oeutil/host/generate_evidence.cpp:228]: Thanks Mingwei! Updated oeutil_quote_provider_log().

**Review by mingweishih — APPROVED** [review]

**Review by CyanDevs — APPROVED** [review]

**Pull request merged**